src/videojs5.hlsjs.js

```diff
@@ -101,6 +101,18 @@
             });
         });
 
+        // Intercept native TextTrack calls and route to video.js directly only
+        // if native text tracks are not supported on this browser.
+        if (!tech.featuresNativeTextTracks) {
+            Object.defineProperty(el, 'textTracks', {
+                value: tech.textTracks,
+                writable: false
+            });
+            el.addTextTrack = function() {
+                return tech.addTextTrack(arguments);
```

Comment:
This is incorrect, as it passes the array of arguments in as a single argument. It should read:

```
return tech.addTextTrack.apply(tech, arguments)
```